src/Mod/Draft/draftguitools/gui_selectplane.py

```diff
@@ -518,9 +518,9 @@ def display(self, arg, icon=None):
         vdir = '('
         vdir += str(_vdir.x)[:4] + ','
         vdir += str(_vdir.y)[:4] + ','
-        vdir += str(_vdir.z)[:4]
-        vdir += ')'
-        vdir = " " + translate("draft", "Dir") + ": " + vdir
+        vdir += str(_vdir.z)[:4] + ')' + ' '
+        vdir *= translate("draft", "Dir", "Dir here means Direction, not Directory. Also shorten the translation because of available space in GUI")
```

Comment:
Are you sure the *= will work here? AFAIR you can only multiply a string by a number....